Explorer/Assets/DCL/Multiplayer/Movement/Settings/RemotePlayerExtrapolationSettings.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+
+namespace DCL.Multiplayer.Movement.Settings
+{
+    public class RemotePlayerExtrapolationSettings : ScriptableObject
```

Comment:
Just saying: that it's not necessary to distribute individual settings to scriptable objects, they could be simply different structures in `MultiplayerMovementSettings`.